docs/advanced/advanced.md

```diff
@@ -278,6 +278,40 @@ After the build, view the html with:
 
 `start docs\_build\html\index.html`
 
+(sphinx:manual-assets)=
+## Manually specify extra files/folders to be included in a website
+
+Jupyter Book will copy over any files that are linked from within its pages so that the links work in the built website. However, sometimes you'd like to manually ensure that files and folders are included in your built website. For example, if you'd like to link to them from _outside_ your built documentation, but not from within your built documentation.
+
+To manually specify items to copy over, use the [`html_extra pash` Sphinx configuration](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path). You can configure this with Jupyter Book liks so:
```

Comment:
Minor: "html_extra pash" looks like a typo and should be "html_extra_path".

Reply:
"liks so" is another typo

Reply:
@choldgraf looks like you need to add a spel checker to your editor 😛 I would suggest https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

We could actually think about adding https://github.com/reviewdog/action-misspell to the CI (with "UK" locale 😉)

Reply:
```suggestion
To manually specify items to copy over, use the [`html_extra path` Sphinx configuration](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path).
You can configure this with Jupyter Book like so:
```

@davidwagner, if you didn't know, you can directly suggest changes like this